### DISCUSSION TITLE
CSS and version update question.

### DISCUSSION THREAD

**Discussion by 54m4d** [Daksh777/SpotifyNoPremium#19] [Q&A] [answered]
Hello Daksh

Is there any difference between `For new UI` and `For update v1.66+`?

I'm only using Spotify 1.1.66 and above, in the user.css can i remove everything before line 75?

**Comment by Daksh777** [comment, marked as answer]
Hey Fosam,

Yes, Spotify completely changed their CSS classes after v1.66+ so if you're not using any older Spotify version, you can remove all the CSS before line 75 and it should be fine

**Reply by 54m4d** [reply]
Great!
And i can also add this css under other themes to get rid of e.g. the download button, right?

**Reply by Daksh777** [reply]
Yes, you can. It should not cause any problems.

**Comment by 54m4d** [comment]
Hello Daksh!
This question is out of topic, what is the latest version of Spotify available in India? i'm asking this becasuse some regions have higher version number than other regions when it comes to updates and i'm certain that India is one of the first regions to receive new updates, so i would like to know the full version info if possible.

**Reply by Daksh777** [reply]
I'm currently on 1.1.67.586.gbb5ef64e

**Reply by 54m4d** [reply]
I switched my location to India but i still get `1.1.66.580.gbd43cbc9-a` both when trying to download the installation file and when trying to update Spotify
Do you know if there'a anything to do other than switching the location to get a version like the one you have?

**Reply by Daksh777** [reply]
I'm unaware of any other method which allows updating the app except the auto-updater.
Also, it doesn't really make a big difference since there are almost no changes in Spotify in the new update (on the user side, of course) and you receive the update in a day or two anyway.

**Reply by 54m4d** [reply]
Yeah, some updates take longer and some don't, i'm just checking which regions are first to receive updates, and why when i switched to India i couldn't get the same version like you.

Could you please download Spotify for Windows and screenshot the version info? (Not from Microsoft Store)

**Reply by Daksh777** [reply]
I did it, and it auto-updated back to the current latest version (1.1.67.586.gbb5ef64e)

**Reply by 54m4d** [reply]
What do you mean by  auto-updated back? Weren't you on `1.1.67.586.gbb5ef64e`?

**Reply by Daksh777** [reply]
I was on `1.1.67.584.gc0d1ad42` (from the Spotify installer) and it auto-updated to `1.1.67.586.gbb5ef64e` after a restart of the app.

**Reply by 54m4d** [reply]
Ah, i see, normally when a new update becomes available the installer gets updated as well according to what i have seen in the past, but maybe in this case there was a slight delay.

Thanks for cheking this.
